modules/rostests/apitests/wlanapi/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 
 add_executable(wlanapi_apitest wlanapi.c testlist.c)
 set_module_type(wlanapi_apitest win32cui)
-add_importlibs(wlanapi_apitest wlanapi msvcrt kernel32)
+# wlanapi is available on WXPsp3 and WVista (and ReactOS), but not on WS03.
+add_delay_importlibs(wlanapi_apitest wlanapi)
```

Comment:
I tend to think the relative instability of delay imports adds more risk than the "soft" failure on 2003 adds as a benefit.

Reply:
Superseded by #208.